### ISSUE TITLE
Individual page ceartion: focus doesn't work properly with radio buttons

### ISSUE THREAD

**Issue by sepidehshahi** [accessibility-in-action/platform#938]
Safari/ Keyboard navigation/ Community connector individual page creation

When tabbing on radio buttons, focus skips the second option and jumps to the next question. For example, "Can you connect to refugees and/or immigrants? (required)" focus goes to "Yes" then on the next tab it jumps to "Yes" option for " Can you connect to people who are marginalized based on gender or sexual identity? (required)" User can only go to the second option by arrow keys.

**Comment by greatislander** [comment]
This is how radio groups are navigated using a keyboard, and is the same for any website using the `<input type"radio" />` element. See description of keyboard controls here: https://www.w3.org/WAI/ARIA/apg/patterns/radiobutton/
